app/assets/javascripts/report-a-problem-form.js

```diff
@@ -79,5 +80,11 @@
     this.$form.trigger('reportAProblemForm.success', data);
   };
 
+  ReportAProblemForm.prototype.trackEvent = function(action){
+    if (GOVUK.analytics && GOVUK.analytics.trackEvent) {
```

Comment:
I am curious about why we need to test for the existence of `GOVUK.analytics` if this is already exercised in the test.